clouditor-engine-aws/src/main/java/io/clouditor/credentials/AwsAccount.java

```diff
@@ -38,16 +43,29 @@
 import software.amazon.awssdk.services.sts.StsClient;
 import software.amazon.awssdk.services.sts.model.StsException;
 
+@Entity(name = "aws_account")
+@Table(name = "aws_account")
 @JsonTypeName(value = "AWS")
 public class AwsAccount extends CloudAccount<AwsCredentials>
-    implements AwsCredentials, AwsCredentialsProvider {
+    implements AwsCredentials, AwsCredentialsProvider, Serializable {
```

Comment:
This is more a general remark. Was there any specific reason why to include `Serializable`. I was looking for information but there was no clear "need" to do it apparently, some recommend it, some say it is not necessary.

Reply:
It is necesarry, becouse trugh the saveOrUpdate or the delete methode, we directly pass one object to be saved or deleted.